episodes/08-quarto.Rmd

```diff
@@ -0,0 +1,122 @@
+    keep-md: true
+source: md
+editor: 
+  markdown: 
+    wrap: 72
+---
+
+```{r}
+#| label: setup
+#| include: false
+source("data/download_data.R")
+```
+
+::: objectives
+-   Create a .qmd document containing R code, text, and plots
+-   Create a YAML header to control output
+-   Understand basic syntax of Quarto and Markdown
+-   Customise code chunks to control formatting
+-   Use code chunks and in-line code to create dynamic, reproducible
+    documents
+:::
+
+::: questions
+-   What is Quarto?
+-   How can I integrate my R code with text and plots?
+-   How can I convert .qmd files to .html?
+:::
+
+## Quarto
+
+Quarto is a flexible type of document that allows you to seamlessly
+combine executable R code, and its output, with text in a single
+document. These documents can be readily converted to multiple static
+and dynamic output formats, including PDF (.pdf), Word (.docx), and HTML
+(.html).
+
+The benefit of a well-prepared Quarto document is full reproducibility.
+This also means that, if you notice a data transcription error, or you
+are able to add more data to your analysis, you will be able to
+recompile the report without making any changes in the actual document.
+
+Quarto comes pre-installed with RStudio (as of v2022.07), so no action
+is necessary.
+
+<figure>
+
+```{r rmarkdown-wizards, echo = FALSE, fig.show = "hold", fig.alt = "R Markdown wizard monsters creating a R Markdown document from a recipe. Art by Allison Horst", out.width="100%"}
+knitr::include_graphics("fig/rmarkdown_wizards.png")
+```
+
+<figcaption>Image credit:
+<a href="https://github.com/allisonhorst/stats-illustrations">Allison
+Horst</a></figcaption>
+
+</figure>
+
+## Other output options
+
+You can convert Quarto to a PDF or a Word document (among others). Put
+`pdf` or `word` in the initial header of the file to indicate the
+desired output format.
+
+```         
+---
+title: "My Awesome Report"
+author: "Emmet Brickowski"
+date: ""
+output: word
+---
+```
+
+::: callout
+## Note: Creating PDF documents
+
+Creating .pdf documents may require installation of some extra software.
+The R package `tinytex` provides some tools to help make this process
+easier for R users. With `tinytex` installed, run
+`tinytex::install_tinytex()` to install the required software (you'll
+only need to do this once) and then when you render to pdf `tinytex`
+will automatically detect and install any additional LaTeX packages that
+are needed to produce the pdf document. Visit the [tinytex
+website](https://yihui.org/tinytex/) for more information.
+:::
+
+::: callout
+## Note: Inserting citations into a Quarto file
+
+It is possible to insert citations into a Quarto file using the editor
+toolbar. The editor toolbar includes commonly seen formatting buttons
+generally seen in text editors (e.g., bold and italic buttons). The
+toolbar is accessible by using the settings dropdown menu (next to the
+'Knit' dropdown menu) to select 'Use Visual Editor', also accessible
```

Comment:
This will be the 'Render' dropdown menu for Quarto

Reply:
Done, thanks!